libraries/botbuilder-azure-blobs/src/blobsStorage.ts

```diff
@@ -26,7 +26,7 @@ export interface BlobsStorageOptions {
 export class BlobsStorage implements Storage {
     private readonly _containerClient: ContainerClient;
     private readonly _concurrency = Infinity;
-    private _initializePromise: Promise<unknown>;
+    private _initializePromise?: Promise<unknown>;
```

Comment:
subtle, but I believe this is a good change.

Reply:
`strict` mode caught it! I wish we could easily turn that on across the board without spending weeks chasing down things like this.